Main/Controllers/CollectionsController.cs

```diff
@@ -47,5 +47,30 @@ public IActionResult AddCollection([FromBody] CollectionCreationData data)
       _dbContext.SaveChanges();
       return Ok();
     }
+
+    [HttpDelete]
+    public IActionResult DeleteCollection([FromBody] int id)
```

Comment:
It's a better practice to pass the is to a delete request via path, not body. Consider changing